i18n/en.pot

```diff
@@ -50,6 +50,29 @@ msgstr ""
 msgid "selected"
 msgstr ""
 
+msgid "All {{count}} items on this page are selected."
+msgid_plural "All {{count}} items on this page are selected."
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "Select all {{count}} items in all pages"
+msgid_plural "Select all {{count}} items in all pages"
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "There are {{count}} items selected in all pages."
+msgid_plural "There are {{count}} items selected in all pages."
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "Clear selection"
+msgstr ""
+
+msgid "There are {{count}} items selected on other pages."
+msgid_plural "There are {{count}} items selected on other pages."
+msgstr[0] ""
+msgstr[1] ""
```

Comment:
We are not creating custom messages for singular/plural, is that deliberate?

The fr/es translations for those keys seem to be missing, after a `yarn update-po` they appear.